bin/ka-clone

```diff
@@ -178,6 +180,48 @@ def _git_dir():
     ).decode('utf-8').rstrip()
 
 
+def backup_existing_hooks():
+    """Backup existing hooks in the git hooks directory."""
+
+    _cli_log_step_success("Backing up existing hooks...")
+
+    hooks_dir = os.path.join(_git_dir(), "hooks")
+
+    # NOTE (Lilli): Why are we doing it this way? Why not just backup each hook
+    # individually? Why not just copy the `hooks` directory? The answer to both
+    # of these questions lies in the way our KA global hooks function. Both the
+    # ka `commit-msg` and `pre-push` hooks search the `hooks` folder for any
+    # other hooks with that start with their name and then runs them. E.g.,
+    # `pre-push` runs all hooks found in the `hooks` folder that start with
+    # `pre-push`, such as `pre-push.lint`. This enables us to have ka hooks,
+    # but still allows the developer to add their own hooks. It's possible for
+    # a user to have previously disabled a global ka hook by renaming the file
+    # to `pre-push.disabled` or `commit-msg.disabled`. Or they may just have
+    # other hooks in there that they don't want to run, that start with
+    # `<global-hook-name>.*`. If we copy the folder, then reinstall the global
+    # hooks, those other hooks will get called. If it was the original ka
+    # `pre-push` or `commit-msg` that was disabled, the developer will get into
+    # infinite loop. (Ask me how I know...) Backing up hooks in the folder
+    # would just get messy and potentially cause the same issue.
+
+    if not os.path.exists(hooks_dir):
+        _cli_log_step_warning("No hooks directory found, skipping backup")
+    else:
+        datetime_str = datetime.datetime.now().strftime("%Y%m%d%H%M%S")
```

Comment:
```suggestion
        datetime_str = datetime.datetime.now().strftime("%Y%m%dT%H%M%S")
```
Suggesting adding the `T` which is a standard way of formatting time at khan, and maybe is more readable.  Totally up to you though.

Reply:
Is the `T` just meant to stick a literal `T` in the middle of the number? Or was it supposed to change the formatting in some particular way?

Reply:
Yes, it's a literal T.  https://medium.easyread.co/understanding-about-rfc-3339-for-datetime-formatting-in-software-engineering-940aa5d5f68a